### PR TITLE
Placement of button issue resolved

### PR DESCRIPTION
**Pull request by Harshi7016**

### CODE REVIEW

**Comment by Utkarshbhimte** [comment]
resolves #39

File: src/components/AddProjectCard/AddProjectCard.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@
   .buttons {
     margin-top: 2rem;
     display: flex;
-    justify-content: flex-end;
+    justify-content: space-between;
```
Comment (Utkarshbhimte) [inline src/components/AddProjectCard/AddProjectCard.scss:28]: this class is used in multiple places. Have you made sure that this is not breaking in other places?

Comment (Harshi7016) [reply src/components/AddProjectCard/AddProjectCard.scss:28]: I have observed in other page where we add blog 
If required Can I go ahead and make this a specific class for the page 
As of  now only Add Blog and Add project uses this particular class and it meets requirements.
I can go ahead and block the scope to particular class by adding different names to class

Comment (Utkarshbhimte) [reply src/components/AddProjectCard/AddProjectCard.scss:28]: > If required Can I go ahead and make this a specific class for the page
yes, that would make sense, having a custom class and have it under a scoped class.

**Pull request merged**